beacon/sync/src/test/java/tech/pegasys/teku/beacon/sync/historical/ReconstructHistoricalStatesServiceTest.java

```diff
@@ -156,6 +168,34 @@ void shouldHandleShutdown(@TempDir final Path tempDir) throws IOException {
     verify(statusLogger, never()).reconstructHistoricalStatesServiceFailedProcess(any());
   }
 
+  @Test
+  void shouldHandleStartWithPartialStorage(@TempDir final Path tempDir) throws IOException {
+    chainBuilder.finalizeCurrentChain();
+    final StorageSystem storageSystem = InMemoryStorageSystemBuilder.buildDefault(spec);
+    final ChainUpdater updater = storageSystem.chainUpdater();
+    updater.initializeGenesis(chainBuilder.getGenesis());
```

Comment:
The genesis block can always be generated from the genesis state, so we can just use:
```suggestion
    updater.initializeGenesis(chainBuilder.getGenesis().getState());
```

and will get the same result without having to add the new chain of methods to initialise genesis with a block and state together.